app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,16 @@
+* {
+  box-sizing: inherit;
+}
+html {
+  margin: 0;
+  padding: 0;
+  font-family: $base-font;
+}
+body {
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+}
+.no-background-color {
+  background-color: rgba(0,0,0,0) !important;
```

Comment:
why not `background-color: none`?

Reply:
only working with the existing one. ``` background-color: none``` not getting the job done. :(

Reply:
solved with use of ```transparent```. ✓ ❤️